design-toolkit/components/src/index.css

```diff
@@ -312,108 +103,108 @@
 @custom-variant disabled { &:is([disabled], [data-disabled]), &:disabled { @slot }}
 
 @utility text-header-xxl {
-  font-size: var(--header-xxl-size);
+  font-size: var(--typography-header-xxl-size);
   font-weight: 500;
   letter-spacing: 0px;
```

Comment:
future todo: font weith and letter spacing can be added to the tokens file and defined there as well